lib/error-handler.js

```diff
@@ -12,10 +26,16 @@ module.exports = {
     }).install();
 
     app.use(Raven.requestHandler());
-    app.use(timeout('10s'));
+
+    const timeoutValue = '10s';
+    app.use(timeout(timeoutValue));
     app.use((req, res, next) => {
       req.on('timeout', () => {
-        req.log.warn('Request timeout');
+        const errorInfo = {
+          ...defaultErrorInfo,
+          isTimeout: true,
+        };
+        req.log.warn(errorInfo, `Request timed out after ${timeoutValue}.`);
```

Comment:
Is it worth responding with a `502` as well to close the connection or do you want the request to continue out?

Reply:
The integration is a little complicated in that regard. For certain things, we immediately respond to the client and just continue processing the request. In other cases, we are working synchronously and for those cases that is an option.
However, error handling is something I'll improve later and this PR should pave the path by allowing better insights.